### PR TITLE
Implementing image detail component

### PR DESCRIPTION
**Pull request by mesparza5**

### CODE REVIEW

File: assets/cms/components/gallery/ImageDetail.vue [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+                <strong>Custom Attributes</strong>
+
+                <p class="image-title">{{props.image.title}}</p>
+                <p class="image-description">{{props.image.description}}</p>
+
+                <div class="mb-4 text-right">
+                    <a :href="props.image.detailUrl" target="_blank" class="btn btn-secondary">Edit Attributes</a>
+                </div>
+            </section>
+            <section v-if="!props.image.displayChoices.lenght">
```
Comment (KorvinSzanto) [inline assets/cms/components/gallery/ImageDetail.vue:18]: Looks like a typo, `length` instead of `lenght`

**Comment by KorvinSzanto** [comment]
Noticed a typo in there, looks good otherwise :+1:

File: assets/cms/components/gallery/ImageDetail.vue (hunk 1 above)
Comment (mesparza5) [reply assets/cms/components/gallery/ImageDetail.vue]: just fixed this typo, thank you @KorvinSzanto :)

**Comment by mesparza5** [comment]
I rebase master by mistake on this PR, I created a new one with fixing the typo and correcting this mistake.

**Pull request closed without merging**